index.phpのcssの追記に関して

mamiko-7172 opened the pull request:
indexのcssを追記してみました。
googleで検証しつつcss記入したので打った内容は間違ってないとは思うのですが、
うまく私のブラウザだと反映されず、みなさんのブラウザでは見えるか確認いただけますか？
よろしくお願いします。

mamiko-7172 committed:
read機能に関して

indexのcssを追記してみました。
googleで検証しつつcss記入したので打った内容は間違ってないとは思うのですが、
うまく私のブラウザだと反映されず、みなさんのブラウザでは見えるか確認いただけますか？
よろしくお願いします。

tai-namechan reviewed (commented):
# cssについて
写真todo感あってめちゃくちゃいいですね！！^^
僕の方だと、作ったデータが横一列に並んで表示されて反映されています！
rowクラスの方にflexwrapを使って折り返しにすると、todo_practiceのような並べ方ができると思います
それとdeleteボタンはボタンタグでcssが反映されているんですけど、editの方がボタンタグじゃないのでdeleteと同じようなinputとbuttonの形にした方がいい気がします

tai-namechan commented on style.css:
```diff
@@ -104,6 +104,58 @@ button:hover {
+    background-color: ivory;
+}
+
+img .index-img{
```
ここなんですけど、僕の方だとimgのサイズのcssが反映されてなくて、img もしくは.index-imgのどっちかで指定しないと反映されない気がします

tai-namechan reviewed (commented):
cssの149行目にもコメントしておきました

mamiko-7172 committed:
read機能のcssに関して追加で修正しました

・EDITのボタン変更
・画像のcssの修正
　→windowsがいけないのか画像の大きさを修正したものが反映されず画像の大きさの修正ができてるか確認お願いします
・rowクラスにflex-wrap追試しました。

よろしくお願いします

chihiro0415 commented:
現状見え方こんな感じですね...！
（横スクロールできちゃう状態）
<img width="900" alt="スクリーンショット 2021-05-03 23 58 10" src="https://user-images.githubusercontent.com/30280895/116893059-7f2bff00-ac6b-11eb-8403-611b674d9c6b.png">

chihiro0415 commented:
[SHOULD]
なめかわさんと同じく、
>作ったデータが横一列に並んで表示されて反映されています！

という状態で、データが5個とかあると3.5個までしか表示されないみたいですね（1.5個分切れてしまっています。。。）

（1番左が表示半分切れてしまっている状態のスクショをはっておきます！）
<img width="900" alt="スクリーンショット 2021-05-04 0 05 20" src="https://user-images.githubusercontent.com/30280895/116893967-81db2400-ac6c-11eb-8e3e-2ac128f94ca7.png">

chihiro0415 commented:
今こんな状態に変わってました！
<img width="900" alt="スクリーンショット 2021-05-04 0 08 33" src="https://user-images.githubusercontent.com/30280895/116894434-0037c600-ac6d-11eb-9c69-ee267bd47765.png">

chihiro0415 reviewed (approved):
コード的に問題はなさそうです！LGTM
（細かいデザインについてはチームにお任せします！）

mamiko-7172 committed:
read機能のcss

追加で修正しました
・cardのmargin追記

chihiro0415 reviewed (approved):
LGTM！
![lgtm](https://user-images.githubusercontent.com/30280895/116896508-4857e800-ac6f-11eb-9820-6e7dcaa561f1.jpeg)

Tomo9996 commented:
先生のおっしゃる通り全部再起動したら私もやっと反映されました！！！
すごく良い感じです！
LGTM！！！
<img width="1440" alt="スクリーンショット 2021-05-04 0 59 57" src="https://user-images.githubusercontent.com/75920419/116901246-a3d8a480-ac74-11eb-8e38-0282a54723f8.png">

Tomo9996 reviewed (approved):
LGTM!!

tai-namechan reviewed (approved):
LGTM!!
